mlflow/server/js/src/model-registry/components/aliases/ModelVersionAliasTag.tsx

```diff
@@ -21,7 +21,6 @@ export const ModelVersionAliasTag = ({
   const { theme } = useDesignSystemTheme();
   return (
     <Tag
-      color='charcoal'
```

Comment:
changed this from 'charcoal' to 'default' to resolve the [X disappearing comment](https://github.com/mlflow/mlflow/pull/10861#issuecomment-1900923138)